### ISSUE TITLE
WCAG 2.2 use of personalization

### ISSUE THREAD

**Issue by rachaelbradley** [w3c/personalization-semantics#202]
In WCAG 2.2 Visible Controls includes a [future technique ](https://www.w3.org/WAI/WCAG22/Understanding/visible-controls.html#techniques).  Looking through the working draft of Personalization against the much updated version of visible controls, I wonder if the markup should be tied to an action.  I would like an action of open menu or display menu but do not see that. Would someone review and advise on the best possible future technique based on the current state of the documents?

**Comment by matatk** [comment]
*Thanks @rachaelbradley. We had time to [discuss this issue very briefly on the Personalization call today](https://www.w3.org/2022/02/07-personalization-minutes.html#t02), and I took an action to update this thread with our understanding of the issue and next steps. This incorporates the feedback that you (@rachaelbradley) gave @JaninaSajka and me via email on the APA chairs list.*

We understand that you're suggesting that we write the proposed future technique "Indicate programmatically that a control is important (future)" and, from our email thread, it sounds like this may incorporate the [`data-simplification`](https://www.w3.org/TR/personalization-semantics-content-1.0/#simplification-explanation) attribute, though the group will discuss this further. As part of our Candidate Recommendation (CR) stage, which is expected imminently, we will be seeking/developing implementations that would support techniques such as this.

As techniques are non-normative, this does not introduce any sort of blocking dependency between the Personalization spec and WCAG, and doesn't need to hold up CR for either of our groups.

We are keen to contribute to WCAG techniques (maybe there are more we could contribute, too) and look forward to working on this.

cc @snidersd @lwolberg @ruoxiran
